.github/workflows/schools_updater.yml

```diff
@@ -0,0 +1,77 @@
+name: Fetch Paginated JSON and Update CSV
+
+on:
+  schedule:
+    - cron: "0 0 * * *"  # Runs once a day at midnight (UTC)
+  workflow_dispatch:    # Allows for manual runs as well
+  push:
+    branches:
+      - aa_schools_csv_updater
+
+jobs:
+  update_csv:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+
+      - name: Install jq and csvkit
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y jq
+          pip install csvkit
+
+      - name: Fetch paginated JSON data from API
+        env:
+          API_URL: ${{ secrets.API_URL }}
+          API_TOKEN: ${{ secrets.API_TOKEN }}
+        run: |
+          # Initialize page and limit parameters
+          PAGE=1
+          LIMIT=250
+          AGGREGATE_FILE=data.json
+          echo "[]" > $AGGREGATE_FILE  # Initialize an empty JSON array
+
+          while true; do
+            # Fetch a page of data from the API
+            RESPONSE=$(curl -s -H "Authorization: Bearer $API_TOKEN" "$API_URL?page=$PAGE&limit=$LIMIT")
```

Comment:
needs to use the `verified[eq]=true` filter, I tried updating it, but it errored for me for some reason.